elasticgraph-indexer_autoscaler_lambda/lib/elastic_graph/indexer_autoscaler_lambda/details_logger.rb

```diff
@@ -30,32 +30,43 @@ def initialize(
         }
       end
 
-      def log_increase(cpu_utilization:, current_concurrency:, new_concurrency:)
+      def log_increase(cpu_utilization:, min_free_storage_in_mb:, current_concurrency:, new_concurrency:)
```

Comment:
I still think that `min` is overloaded (since it can refer both to the threshold at which we pause indexing or the value we compare to that threshold). It's not very clear which this is.  Can you include both (they both seem useful to log) and name them appropriately, similar to the renaming you did in the lambda logic? e.g. `lowest_node_free_storage_in_mb` and `required_free_storage_in_mb`.

(And please apply that throughout this file.)